src/main/kotlin/com/vanniktech/maven/publish/legacy/Pom.kt

```diff
@@ -0,0 +1,39 @@
+package com.vanniktech.maven.publish.legacy
+
+import com.vanniktech.maven.publish.baseExtension
+import com.vanniktech.maven.publish.findOptionalProperty
+import org.gradle.api.Project
+
+internal fun Project.configurePom() {
+  // without afterEvaluate https://github.com/gradle/gradle/issues/12259 will happen
+  afterEvaluate {
```

Comment:
That Gradle bug is kind of terrible for us. It means anyone using the new POM API on a Gradle plugin will need an afterEvaluate.

Reply:
After writing this I realised that it probably makes sense to change the code below to to null checks before calling set to avoid writing over explicit API calls with null